openvdb/openvdb/points/PointAttribute.h

```diff
@@ -67,7 +67,7 @@ inline void appendAttribute(PointDataTreeT& tree,
 /// @param transient          mark attribute as transient
 template <typename ValueType,
           typename CodecType = NullCodec,
-          typename PointDataTreeT = PointDataTree>
+          typename PointDataTreeT>
```

Comment:
I guess we're not calling this with <=1 template args, but I'd leave this defaulted

Reply:
It is a bit surprising, I had thought that all template parameters had to be default from the first one you make default onwards. It seems that's not the case for function template where the template type can be deduced.